docs/content/develop/test/test.md

```diff
@@ -272,6 +272,51 @@ func TestSignatureAlgorithmDiffSuppress(t *testing.T) {
 }
 ```
 
+## Skipping tests in VCR mode
+
+As described above, acceptance tests can be run in VCR modes where HTTP requests are either recorded or are replayed. This system can fail in some scenarios, so we will oftern flag tests that are incompatible with VCR so that they're skipped. This means the tests will not run automatically on PRs (in VCR mode) but the tests will still run in our nightly test suite.
+
+### When should tests be skipped in VCR mode?
```

Comment:
style: [conceptual headings should use noun phrases](https://developers.google.com/style/headings)

```suggestion
### Reasons that tests are skipped in VCR replaying mode
```